torch/csrc/jit/script/lexer.h

```diff
@@ -83,7 +83,9 @@ namespace script {
   _(TK_FOR, "for", "for")                        \
   _(TK_IN, "in", "in")                           \
   _(TK_STARRED, "starred", "")                   \
-  _(TK_UNARY_MINUS, "unary minus", "")
+  _(TK_UNARY_MINUS, "unary minus", "")           \
+  _(TK_POW, "pow operator", "**")                \
+  _(TK_MATMUL, "matmul operator", "@")
```

Comment:
For single characters like "@" our idiom is to put it into valid_single_char_tokens and use '@' in place of TK_MATMUL. See how '*' and '/' are defined.